data/schema/filters/abilities.cfg

```diff
@@ -19,3 +17,11 @@
 	{DEFAULT_KEY type_value value_type empty}
 	{FILTER_BOOLEAN_OPS abilities}
 [/tag]
+
+[tag]
+	name="$filter_abilities"
+	max=0
+	super="$filter_abilities_generic"
+	{SIMPLE_KEY tag_name string_list}
+	{SIMPLE_KEY overwrite_specials string_list}
```

Comment:
Why does this take `string_list` instead of `ability_overwrite`?

Reply:
and why active_on take string_list then what only value must be"offense,defense,both"?

Reply:
`active_on` should probably only take an `ability_context` too

Reply:
in filter? Ok.

Reply:
Just define a list type based on `ability_context`: `{LIST_TYPE ability_context}`

That defines a new `ability_context_list` type.

If there's no need for the non-list version of the type, you can define a list that's not based on an existing type with `LIST_TYPE_COMPLEX`:

```
{LIST_TYPE_COMPLEX ability_context
[element]
# definition of the list element goes here
[/element]
}```

Reply:
LIST_TYPE_COMPLEX seems to me more suitable for filtering ranges of numbers than for a list of characters (both, offense and defense for active_on)